RF_ASM/Utils.cs

```diff
@@ -52,5 +52,13 @@ public static void DisplayHexDump(byte[] data)
                 Console.WriteLine();
             }
         }
+
+        public static void PadBytes(int desiredLength, byte[] ba)
```

Comment:
This does nothing, since you are not passing a pointer to the bytes, and you are not returning anything.

Reply:
I deleted the function and copied SetBitInByte to RF_ASM.Utils... The function was not used... seriously why did you make 3 separate conversations for this...

Reply:
Also reassigning values in that array actually does reassign them everywhere because you're passing the same byte array around... Prepending still doesn't effect the base array though.